src/command_modules/azure-cli-sql/tests/test_sql_commands.py

```diff
@@ -289,6 +289,37 @@ def test_sql_db_mgmt(self, resource_group, resource_group_location, server):
                  .format(rg, server, database_name),
                  checks=[NoneCheck()])
 
+        
+class AzureActiveDirectoryAdministratorScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    @SqlServerPreparer()
+    def test_aad_admin(self, resource_group, server):
+        rg = resource_group
+        sn = server
+        sid = '5e90ef3b-9b42-4777-819b-25c36961ea4d'
+        tid = '72f988bf-86f1-41af-91ab-2d7cd011db47'
+        user = 'DSEngAll'
```

Comment:
This is not a new feature, so we shouldn't expect major changes after the initial commit. @jaczhan , can you create and then later delete an AD service principal in test setup? e.g. I think `az ad app create` should work.